docs/dev/setup.rst

```diff
@@ -257,6 +258,127 @@ You can stop the service using ``sudo service artemis stop`` and restart it usin
 
 Logs can be fetched using ``sudo journalctl -u artemis -f -n 200``.
 
+Run the server via Docker
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Artemis provides a docker image named ``ghcr.io/ls1intum/artemis:<VERSION>``.
+The current develop branch will be deployed as `latest` version.
+Releases like ``5.7.1`` are deployed as ``ghcr.io/ls1intum/artemis:5.7.1``.
+The easiest way to configure a local deployment via docker is a deployment with a docker-compose file.
+You could use a compose file similar to this (as an example this deployment uses the gitlab-jenkins configuration of artemis:
+
+::
+
+    version: '3'
+
+    services:
+      gitlab:
+        image: gitlab/gitlab-ce
+        restart: unless-stopped
+        volumes:
+          - ./data/gitlab/config:/etc/gitlab
+          - ./data/gitlab/logs:/var/log/gitlab
+          - $BACKUP_DIR/gitlab:/var/opt/gitlab/backups # Optional but useful
+          - ./data/gitlab/data:/var/opt/gitlab
+        environment:
+          - GITLAB_OMNIBUS_CONFIG: |
+              external_url "https://${GIT_SERVER_NAME}"
+              nginx['listen_port'] = 80
+              nginx['listen_https'] = false
+              nginx['hsts_max_age'] = 0
+              prometheus_monitoring['enable'] = false
+              gitlab_rails['monitoring_whitelist'] = ['0.0.0.0/0']
+              gitlab_rails['gitlab_username_changing_enabled'] = false
+              gitlab_rails['gitlab_default_can_create_group'] = false
+              gitlab_rails['gitlab_default_projects_features_issues'] = false
+              gitlab_rails['gitlab_default_projects_features_merge_requests'] = false
+              gitlab_rails['gitlab_default_projects_features_wiki'] = false
+              gitlab_rails['gitlab_default_projects_features_snippets'] = false
+              gitlab_rails['gitlab_default_projects_features_builds'] = false
+              gitlab_rails['gitlab_default_projects_features_container_registry'] = false
+              gitlab_rails['backup_keep_time'] = 604800
+        ports:
+          - "${GITLAB_SSH_PORT}:22"
+          - "${GITLAB_HTTP_PORT}:80"
+        networks:
+          - artemis-net
+
+      jenkins:
+        image: jenkins/jenkins:lts
+        restart: unless-stopped
+        user: root
+        volumes:
+          - $BACKUP_DIR/jenkins:/var/jenkins_backup
+          - ./data/jenkins/home:/var/jenkins_home
+          - /var/run/docker.sock:/var/run/docker.sock
+          - /usr/bin/docker:/usr/bin/docker:ro
+          - /usr/bin/com.docker.cli:/usr/bin/com.docker.cli:ro
```

Comment:
I’ve never seen this binary. In which setups does it exist?

Reply:
Nice finding. I think com.docker.cli is some really old content :) 
```suggestion
```